File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.java
class Solution {
    public int longestConsecutive(int[] nums) {
        int res = 0;
        HashSet<Integer> hS = new HashSet<>();
        for(int j=0; j<nums.length; j++){
            hS.add(nums[j]);
        }
        for(int i : hS){
            if(!hS.contains(i-1)){
                int currCount = 0;
                while(hS.contains(i+currCount)){
                    currCount++;
                }
                res = Math.max(res, currCount);
            }
            
        }
        return res;
    }
}

File: Data Structures & Algorithms/maximum-subarray/submission-1.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:
        # Kadane's algo

        res = min(nums)
        cur = min(nums)
        for num in nums:
            if cur < 0:
                cur = 0
            cur += num
            res = max(res, cur)
        return res